frontend/src/components/common/player/player.tsx

```diff
@@ -0,0 +1,124 @@
+import AudioPlayer from 'react-h5-audio-player';
+import 'react-h5-audio-player/lib/styles.css';
+import styles from './styles.module.scss';
+
+type Props = {
+  src: string;
+  onClickPrevious?: () => void;
+  onClickNext?: () => void;
+};
+
+const MILLISECONDS_IN_SECOND = 1000;
```

Comment:
```suggestion
const MILLISECONDS_IN_SECOND = 1000;
```
move to `frontend/src/components/common/player/common/constants.ts`

Reply:
This looks like a global constant, it would be better to have it in shared constants module